src/components/LayerSwitcher/helpers.tsx

```diff
@@ -1,53 +1,15 @@
 import React from 'react';
 import { Box, IconButton, Typography } from '@material-ui/core';
-import Button from '@material-ui/core/Button';
 import CloseIcon from '@material-ui/icons/Close';
 import ListItemIcon from '@material-ui/core/ListItemIcon';
 import List from '@material-ui/core/List';
 import styled from 'styled-components';
-import { useMapStateContext } from '../utils/MapStateContext';
+import { View, useMapStateContext } from '../utils/MapStateContext';
 import { t, Translation } from '../../services/intl';
 
-let counter = 0;
-const TMS_EXAMPLES = [
-  'https://{s}.tile.openstreetmap.fr/hot/{z}/{x}/{y}.png',
-  'https://stamen-tiles.a.ssl.fastly.net/watercolor/{z}/{x}/{y}.jpg',
-];
-
-export const AddUserLayerButton = ({ setUserLayers }) => {
-  const { setActiveLayers } = useMapStateContext();
-  const onClick = () => {
-    const TMS_EXAMPLE = TMS_EXAMPLES[counter % TMS_EXAMPLES.length];
-    counter += 1;
-    const url = prompt(t('layerswitcher.add_layer_prompt'), TMS_EXAMPLE); // eslint-disable-line no-alert
```

Comment:
As this PR removes the ability to submit custom URL, can we perhaps add `Custom URL` as a first option in the Autocomplete dropdown and display the input below?

Followup is ok. Let me know :)